sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -66,17 +66,24 @@ export class AssessmentAESTController extends BaseController {
       })),
     ];
 
-    // submitted date sorting
-    requestedAssessments = requestedAssessments.sort((first, second) =>
-      first.submittedDate > second.submittedDate ? 1 : -1,
-    );
-    // sorting with status and returning
-    return requestedAssessments.sort((first) =>
-      first.status === StudentAppealStatus.Pending ||
-      first.status === ScholasticStandingStatus.Pending
-        ? -1
-        : 1,
-    );
+    // status and submitted date sorting
+    return requestedAssessments.sort((first, second) => {
+      if (
+        first.status === second.status &&
+        first.submittedDate > second.submittedDate
+      ) {
+        return -1;
+      }
+      if (
+        (first.status === StudentAppealStatus.Pending &&
```

Comment:
This is could also be simplified as `first.status !== second.status`, right?

Reply:
no here, the first status should be pending and second declined, not the reverse